packages/webapp/src/pages/Environment/Settings/Show.tsx

```diff
@@ -52,80 +43,80 @@ export const EnvironmentSettings: React.FC = () => {
         element.scrollIntoView({ behavior: 'smooth' });
     }, [environmentAndAccount]);
 
-    const handleDelete = async () => {
-        const { res } = await apiDeleteEnvironment(env);
-        if (res.status >= 200 && res.status < 300) {
-            setShowDeleteAlert(false);
-            // We have to start by changing the url, otherwise PrivateRoute will revert the env based on it.
-            navigate(`/${PROD_ENVIRONMENT_NAME}/environment-settings`);
-            await mutateMeta();
-            setEnv(PROD_ENVIRONMENT_NAME);
-            toast({
-                title: 'The environment has been deleted successfully',
-                variant: 'success'
-            });
-        } else {
-            toast({
-                title: 'Failed to delete environment',
-                variant: 'error'
-            });
-        }
-    };
-
     if (!environmentAndAccount || !team) {
         return (
-            <DashboardLayout className="p-6">
+            <DashboardLayout fullWidth className="flex-col w-[988px] 4xl:w-full px-0 4xl:px-[208px]">
```

Comment:
Tailwind is mobile first, so this means `w-[998px] px-0`, unless screen is larger than 4xl, then `w-full` and `px-[208px]`. Is it intentional? I find it weird when I shrink the screen and the content sticks to the border. And it's pretty different behavior than the other new screens. We can iterate on it, but I'd make them behave the same at least.

I tried out the default behavior and it looks pretty similar tbh, but without the problems:
```suggestion
            <DashboardLayout className="flex-col">
```

Reply:
Though I think we need this screen to just be wider in general, specially for the Connect UI section.

Reply:
This is the responsive behaviour proposed by Gaby https://www.figma.com/design/a4t7gaUATMqUcWedowFPFd/%F0%9F%90%BA-Design-Sytem---Product?node-id=6939-19029&m=dev